### PR TITLE
isVotedOn and isDead

### LINKED ISSUE #23: voting -> colours? counts?
**Easier:** When you vote, it'd be nice to at least see the one you voted for is highlighted. Maybe with `background: 'red'` This would at least visually indicate to the user that they (already) voted for someone, without having to click something to find out if their action did something.

**Harder:** Also nice-to-have would be if the count of votes showed up, and/or even who voted for who, like in the game "Among us".

### PR DESCRIPTION
**Pull request by hchiam**
and votedPlayers and .wobble-to-rest and .wobble-indefinitely

To resolve #23.

## if voted on: -> yellow, wobble for a little bit, subtle shadow:

voted-animation

and

## if not alive: -> pink and upside down

I can't simulate the votes, so I just tried swapping the conditional to test how it looks:

not-alive

### CODE REVIEW

File: mafia-react/src/index.css [hunk 1]
```diff
@@ -85,3 +85,32 @@ body[data-is-night='false'] [data-is-user='false'] .card {
...
+@keyframes rest {
+  100% {
+    transform: rotate(0deg);
+  }
+}
+
+.wobble-indefinitely {
+  transform: rotate(0);
+  animation: wobble forwards infinite 1s;
+}
```
Comment (hchiam) [inline mafia-react/src/index.css:110]: `.wobble-indefinitely` isn't used right now, but this is available if we want it to keep something wobbling. I can delete it if you'd like, to remove dead code.

**Comment by alan-mj-lin** [comment]
Do you think maybe upside down will be a little confusing to depict someone who's out of the game? The red shade is nice though.  Idk what do you think?

**Comment by hchiam** [comment]
> Do you think maybe upside down will be a little confusing to depict someone who's out of the game? The red shade is nice though. Idk what do you think?

Up to you. Just playing with the idea since it's easy to remove. I'll remove the transform for now. 

After doing work in a11y, I always try to make indicators that don't rely on colour alone. Having it upside-down is playful, but I guess we could change the icon to have an X instead. Or make just the icon upside down. I'd also like to hide the buttons for players that aren't alive. I'll try another option. Let me know what you think

**Comment by hchiam** [comment]
> Do you think maybe upside down will be a little confusing to depict someone who's out of the game? The red shade is nice though. Idk what do you think?

Up to you. Just playing with the idea since it's easy to remove. I'll remove the transform for now.

After doing work in a11y, I always try to make indicators that don't rely on colour alone. Having it upside-down is playful, but I guess we could change the icon to have an X instead. Or make just the icon upside down. I'd also like to hide the buttons for players that aren't alive. Let me know what you think. 
## In day:
Oct-05-2020 18-33-38
## At night:
Screen Shot 2020-10-05 at 18 35 41

**Pull request merged**